dotnet/src/Connectors/Connectors.AzureAIInference/Core/ChatClientCore.cs

```diff
@@ -289,7 +289,6 @@ internal async IAsyncEnumerable<StreamingChatMessageContent> GetStreamingChatMes
                 StreamingChatCompletionsUpdate update = responseEnumerator.Current;
                 metadata = GetResponseMetadata(update);
                 streamedRole ??= update.Role;
-                streamedName ??= update.AuthorName;
```

Comment:
It seems the `streamedName` variable can be dropped as well because it's not used anywhere after the change.